### ISSUE TITLE
[Fair Map] Possible zoom related improvements.

### ISSUE THREAD

**Issue by alloy** [artsy/eigen#154]
These are just a few things I came across and would like to get all your’s yay or nay on.

- [ ] The first double-tap only zooms in 1 level, but doesn’t yet show any extra features. I think it could, because it doesn’t take much more zooming to make them show or possibly the 1st level could zoom in a bit more?

![ios simulator screen shot 11 feb 2015 21 21 07](https://cloud.githubusercontent.com/assets/2320/6155472/012970ca-b234-11e4-8034-6311775f2356.png)

- [ ] When zoomed-in, it’s easily possible to scroll the content out of view in all corners. Should we constrain that?

![ios simulator screen shot 11 feb 2015 21 22 23](https://cloud.githubusercontent.com/assets/2320/6155491/18c21e3a-b234-11e4-9233-4b9b2d7887c6.png)

**Comment by orta** [comment]
TLDR: I think the first bit should be done :+1: - and the second not.

Both of these ideas came from the fact that we wanted the map to be generic and to support whatever we are given ( The map renderer is the same as an artwork renderer for example. ) But also that we expect to support a wide array of maps ( not all will be at this 45' angle. )
